d2l-table-style.html

```diff
@@ -4,7 +4,7 @@
 <dom-module id="d2l-table-style">
 	<template>
 		<style include="d2l-colors">
-			:host .table-container > ::content > table {
+			:host .table-container > ::content table[is="d2l-table"] {
```

Comment:
These selectors had to change because the `table` is no longer the direct descendent and you needed something more specific?